lib/patches/bundler/bundler/current_ruby.rb

```diff
@@ -6,5 +6,9 @@ class CurrentRuby
     def truffleruby?
       defined?(RUBY_ENGINE) && RUBY_ENGINE == "truffleruby"
     end
+
+    def ruby?
+      truffleruby?
```

Comment:
Should it be
```ruby
!mswin? && (!defined?(RUBY_ENGINE) || RUBY_ENGINE == "ruby" || RUBY_ENGINE == "rbx" || RUBY_ENGINE == "maglev" || RUBY_ENGINE == "truffleruby")
```
to match upstream more closely?
I guess it doesn't matter much but still.

Reply:
I originally aliased the method and did something like:

```
def ruby?
  truffleruby? || original_ruby?
end
```

But, since the patches only load in TruffleRuby all of those other checks were guaranteed to fail anyway.

Reply:
Fair enough.

Reply:
Yeah that would have been better, I did not realized this overrides and loses the original behavior. We might use our code to test something on other ruby and get a nasty surprise, @nirvdrum could you change it even though this got already merged.

Reply:
Never-mind :)

Reply:
Well, I opened #849 if you guys feel strongly about it. It doesn't bother me much either way.